docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+
+Configure Options on Tasks When Running a Subflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify options on tasks in subflows with the following syntax:
+
+.. code-block:: yaml
+
+    <flow_to_modify>:
+        steps:
+            <step_number>:
+                ``flow``: ``<sub_flow_name>``
+                options:
+                    ``<task>``:
+                        ``<option_name>``: ``<value>``
+
+Replace all values with ``<>`` with the desired values.
+
+    Example: Let's examine the definition of the ``ci_master`` flow from the universal ``cumulusci.yml`` file:
+
+.. code-block::
+
+    ci_master:
+        group: Continuous Integration
+        description: Deploy the package metadata to the packaging org and prepare for managed package version upload.  Intended for use against main branch commits.
+        steps:
+            1:
+                flow: dependencies
+                options:
+                    update_dependencies:
+                        include_beta: False
+            2:
+                flow: deploy_packaging
+            3:
+                flow: config_packaging
+
+This flow specifies that when the subflow ``dependencies`` runs, the ``include_beta`` option is passed a value of ``False`` to the ``update_dependencies`` task (which itself executes in the ``dependencies`` subflow). 
+
+
+``when`` Clauses
+^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a ``when`` clause in a flow step to conditionally run that step. A ``when`` clause is written in a Pythonic syntax that can evaluate to a boolean (``True`` or ``False``) result. 
+
+The variables that are available for reference in ``when`` clauses [TO BE ADDED]
+
+A common use case is to be able to check [TO BE ADDED]
+
+See `use variables for task options`_ for more information.
+
+
+
+Scratch Org Configurations
+--------------------------
+
+This section defines the scratch org configurations that are available without explicitly running ``cci org scratch`` to create a new configuration. If you're looking for more information on the use of scratch orgs with CumulusCI, see `Scratch Org Environments<TODO>`_. 
+
+
+Override Default Values
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note:: These overrides pertain only to scratch orgs.
+
+You can override these values for your org:
+
+* ``days`` (integer): Number of days for the scratch org to persist.
+* ``namespaced`` (boolean): Whether or not the scratch org is a `namespaced org<TODO sf link?>`_.
+* ``config_file`` (string): Path to the org definition file to use when building the scratch org.
+
+.. code-block:: yaml
+    
+    orgs:
+        scratch:
+            ``<org_name>``:
+                ``<key>``: ``<value>``
+
+Replace all values with <> with the desired values.
+
+    Example: Override the default number of days from 7 to 15 on the ``dev`` org.
+
+.. code-block:: yaml
+
+    orgs:
+        dev:
+            days: 15
+
+
+
+Configuration Scopes
```

Comment:
@davisagli, @Br4nd0R, @davidmreed: There might need to be more discussion about how to fully convey configuration scopes. I think the thorniest issue is ``Local Project`` and how it differs from ``Project``.

Reply:
@justinvholt - I think we should note these "thorny" sections and potentially get a list going for candidates for an "Advanced" section. I think that this would be something we could address after the beta release of the docs. Our team hasn't received many questions on this topic, and this is probably something only 1% users are currently taking advantage of.

Reply:
Noted in the CCI Docs "Items to Do" list.